common/static/coffee/spec/discussion/view/new_post_view_spec.coffee

```diff
@@ -10,21 +10,29 @@ describe "NewPostView", ->
         )
         @discussion = new Discussion([], {pages: 1})
 
-    checkVisibility = (view, expectedVisible, expectedDisabled) =>
-      view.render()
-      expect(view.$(".js-group-select").is(":visible")).toEqual(expectedVisible)
-      disabled = view.$(".js-group-select").prop("disabled")
-      if expectedVisible and ! expectedDisabled
+    checkVisibility = (view, expectedVisible, expectedDisabled, noRender) =>
+      if !noRender
+        view.render()
+      # Can also be undefined if the element does not exist.
+      expect(view.$('.group-selector-wrapper').is(":visible") or false).toEqual(expectedVisible)
```

Comment:
@singingwolfboy If the element does not exist, it cannot be visible. In most cases that it wouldn't be visible, it would be undefined, as this is usually when no 'cohort_options' dictionary is passed. However if cohort menu is there and is just hidden, I wouldn't consider that a failure. If someone changes the mechanism by which the menu is taken from view, this test will be able to handle it in a duck-typey manner.